programs/clearing_house/src/controller/orders.rs

```diff
@@ -248,14 +248,15 @@ pub fn place_order(
     let risk_decreasing = worst_case_base_asset_amount_after.unsigned_abs()
         <= worst_case_base_asset_amount_before.unsigned_abs();
 
-    let meets_initial_maintenance_requirement =
-        meets_initial_margin_requirement(user, perp_market_map, spot_market_map, oracle_map)?;
-
-    if !meets_initial_maintenance_requirement && !risk_decreasing {
-        return Err(ErrorCode::InsufficientCollateral);
-    }
+    let meets_initial_margin_requirement = meets_place_order_margin_requirement(
```

Comment:
right now you can place_order when market isn't active

Reply:
think its ok to queue orders until its active